scripts/entrypoint.sh

```diff
@@ -7,16 +7,14 @@ figlet -t "Kartoza Docker GeoServer"
 source /scripts/functions.sh
 source /scripts/env-data.sh
 
-/bin/bash /scripts/start.sh
+set_vars
+export  READONLY CLUSTER_DURABILITY BROKER_URL EMBEDDED_BROKER TOGGLE_MASTER TOGGLE_SLAVE BROKER_URL
+export CLUSTER_CONFIG_DIR MONITOR_AUDIT_PATH CLUSTER_LOCKFILE INSTANCE_STRING
```

Comment:
The instance string is always going to be empty like the logs are showing
![empty](https://user-images.githubusercontent.com/2510900/152691646-febb00d5-4f2b-413e-8d6c-d6d0f3a40d75.png)